hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/hapi/ContractCallSuite.java

```diff
@@ -243,7 +246,25 @@ public List<HapiSpec> getSpecsInSuite() {
                 nestedContractCannotOverSendValue(),
                 depositMoreThanBalanceFailsGracefully(),
                 lowLevelEcrecCallBehavior(),
-                callsToSystemEntityNumsAreTreatedAsPrecompileCalls());
+                callsToSystemEntityNumsAreTreatedAsPrecompileCalls(),
+                hollowCreationFailsCleanly());
+    }
+
+    private HapiSpec hollowCreationFailsCleanly() {
+        final var contract = "HollowAccountCreator";
+        return onlyDefaultHapiSpec("HollowCreationFailsCleanly")
```

Comment:
I don't think `onlyDefaultHapiSpec` is right?

Reply:
True, removed!